packages/zowe-explorer/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
+- Implemented sorting of PDS members by date created [#2707](https://github.com/zowe/vscode-extension-for-zowe/pull/2707)
```

Comment:
Is there an equivalent issue we can link to instead of the pull request for v2?

Reply:
https://github.com/zowe/vscode-extension-for-zowe/issues/2565